joss-paper/paper.md

```diff
@@ -0,0 +1,159 @@
+---
+title: 'PyFstat: a Python package for continuous gravitational-wave data analysis'
+tags:
+  - Python
+  - gravitational waves
+  - continuous waves
+  - pulsars
+  - data analysis
+authors:
+  - name: David Keitel
+    orcid: 0000-0002-2824-626X
+    affiliation: "1"
+  - name: Rodrigo Tenorio
+    orcid: 0000-0002-3582-2587
+    affiliation: "1"
+  - name: Gregory Ashton
+    orcid: 0000-0001-7288-2231
+    affiliation: "2"
+  - name: Reinhard Prix
+    orcid: 0000-0002-3789-6424
+    affiliation: "3, 4"
+affiliations:
+  - name: Departament de Física, Institut d'Aplicacions Computacionals i de Codi Comunitari (IAC3), Universitat de les Illes Balears, and Institut d'Estudis Espacials de Catalunya (IEEC), Crta. Valldemossa km 7.5, E-07122 Palma, Spain
+    index: 1
+  - name: OzGrav, School of Physics & Astronomy, Monash University, Clayton 3800, Victoria, Australia
+    index: 2
+  - name: Max-Planck-Institut für Gravitationsphysik (Albert-Einstein-Institut), D-30167 Hannover, Germany
+    index: 3
+  - name: Leibniz Universität Hannover, D-30167 Hannover, Germany
+    index: 4
+date: 08 January 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Gravitational waves in the sensitivity band of ground-based detectors
+can be emitted by a number of astrophysical sources,
+including not only binary coalescences, but also individual spinning neutron stars.
+The most promising signals from such sources,
+although as of 2020 not yet detected,
+are the long-lasting, quasi-monochromatic 'Continuous Waves' (CWs).
+Many search methods have been developed and applied on
+LIGO [@TheLIGOScientific:2014jea]
+and Virgo [@TheVirgo:2014hva] data.
```

Comment:
```suggestion
Advanced LIGO [@TheLIGOScientific:2014jea]
and Advanced Virgo [@TheVirgo:2014hva] data.
```

Reply:
While the references cover only Advanced L/V, the sentence refers to the initial era as well. We could add initial-detector refs (:-1:) or maybe
```suggestion
(Advanced) LIGO [@TheLIGOScientific:2014jea]
and Virgo [@TheVirgo:2014hva] data.
```
?

Reply:
Too much clutter I'd say: leave as is